modules/document_repository/ajax/documentDelete.php

```diff
@@ -31,40 +36,58 @@
 $DB =& Database::singleton();
 
 $Notifier = new NDB_Notifier(
-    "document_repository",
-    "delete"
+    'document_repository',
+    'delete'
 );
 
 $rid = $_POST['id'];
 
 $fileName = $DB->pselectOne(
-    "Select File_name from document_repository where record_id =:identifier",
+    'SELECT File_name FROM document_repository WHERE record_id =:identifier',
     array(':identifier' => $rid)
 );
 $userName = $DB->pselectOne(
-    "Select uploaded_by from document_repository where record_id =:identifier",
+    'SELECT uploaded_by FROM document_repository WHERE record_id =:identifier',
     array(':identifier' => $rid)
 );
 $dataDir  = $DB->pselectOne(
-    "Select Data_dir from document_repository where record_id =:identifier",
+    'SELECT Data_dir FROM document_repository WHERE record_id =:identifier',
+    array(':identifier' => $rid)
+);
+$version  = $DB->pselectOne(
```

Comment:
Is `$version` being used anywhere else in the file? What is the purpose for querying the version?

Reply:
Thanks! I forgot to remove from old code logic.